astro/src/content/blog/fusionauth-inc-5000-2024.mdx

```diff
@@ -0,0 +1,40 @@
+---
+publish_date: 2024-08-13
+title: FusionAuth Named to the Inc. 5000 (Again)!
+description: FusionAuth is proud to be named to the Inc. 5000 list of the fastest-growing companies in the United States. 
+authors: FusionAuth
+image: /img/blogs/inc-5000-2024/inc5000-2024.png
+categories: News
+tags: fusionauth, inc 5000, growth
+excerpt_separator: "{/* more */}"
+---
+
+We're thrilled to announce that FusionAuth has once again made it onto the prestigious Inc. 5000 list of fastest-growing companies. We climbed the ladder this year, going from #1597 to #1269. The Inc. 5000 list features some of the fastest-growing, private businesses in the US. Inc. calculates the ratings based on the growth from the previous three years.
+
+{/* more */} 
+
+Outside of the Inc. ranking, a key component that we track is growth based on company size. Companies at our same size on average grow roughly 30% per year according to [Eleken](https://www.eleken.co/blog-posts/average-saas-growth-rate-brief-guide-for-startups) FusionAuth grew over 65% last year and over 130% the year before that. And wait it gets even better. In 2021, we grew 314%! And we have no plans on slowing down either.
+
+## Still Growing
+
+We've been hiring like mad in every department since last year. The investment we received  from [Updata Partners](/blog/fusionauth-funding) last year allowed us to hire the right people for the right roles...right now. 
+
+Oh yeah and by the way, [we're still hiring](/jobs), because we've been busier than ever over the past year. Here's a peek behind the curtain: 
+
+## Simplifying CIAM
+
+Developers from around the world have downloaded FusionAuth over 20 million times. In a world where everything is becoming multi-tenant, nickel-and-dime pricing, it's refreshing to be different. FusionAuth is still the only commercially-supported customer login solution that you can install anywhere, even air-gapped!
+
+One challenge that we've had to face is that our unique position brings complexity with it. We've listened to our users and we've decided to spend the next few release cycles simplifying FusionAuth in every way possible. That started with our [Simple Theme builder](/blog/announcing-fusionauth-1-51) in version 1.51.0. Over the next few months, you'll see FusionAuth become even easier to use, no matter what your use case.
```

Comment:
```suggestion
Because we serve both self-hosted and cloud users, and because auth is complex, we've built a product that is powerful but can be overwhelming at first glance. We've listened to our users and decided to spend the next few release cycles simplifying FusionAuth to make that first experience and common operations better. That started with our [Simple Theme builder](/blog/announcing-fusionauth-1-51) in version 1.51.0. Over the next few months, you'll see FusionAuth become even easier to use, no matter what your use case.
```